app/src/controllers/project/selectors.js

```diff
@@ -22,7 +23,19 @@ export const projectPreferencesSelector = (state) => projectSelector(state).pref
 
 export const userFiltersSelector = (state) => projectPreferencesSelector(state).filters || [];
 
-export const defectTypesSelector = (state) => projectConfigSelector(state).subTypes || {};
+export const defectTypesSelector = (state) => {
```

Comment:
Please use `createSelector` to avoid unnecessary recalculations.

Reply:
Done